docs/user/openstack/customization.md

```diff
@@ -192,7 +192,9 @@ controlPlane:
       - fa806b2f-ac49-4bce-b9db-124bc64209bf
 ```
 
-**NOTE:** Allowed address pairs won't be created for the additional networks.
+**NOTES:**
```

Comment:
Can you also update the note in the "Machine Pools" section, on line 48 of this file?

https://github.com/openshift/installer/pull/4551/files#diff-8d13bb62fd33c601f7f64e3a9c861445f547962a0079d20613daab0c07dd73b8R48